src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -1,236 +1,126 @@
-          expect(form.submit).to.be.a('function');
-        });
-      }
-
-      if (form.savedFormMessages) {
-        it('should have savedFormMessages', () => {
-          expect(form.savedFormMessages).to.be.a('object');
-        });
-      }
-
-      if (form.transformForSubmit) {
-        it('should have transformForSubmit', () => {
-          expect(form.transformForSubmit).to.be.a('function');
-        });
-      }
-
-      it('should have confirmation', () => {
-        expect(form.confirmation).to.be.a('function');
-      });
-
-      if (form.preSubmitInfo) {
-        it('should have preSubmitInfo', () => {
-          expect(form.preSubmitInfo).to.be.a('object');
-        });
-      }
-
-      if (form.footerContent) {
-        it('should have footerContent', () => {
-          expect(form.footerContent).to.be.a('function');
-        });
-      }
-
-      if (form.getHelp) {
-        it('should have getHelp', () => {
-          expect(form.getHelp).to.be.a('function');
-        });
-      }
-
-      if (form.errorText) {
-        it('should have errorText', () => {
-          expect(form.errorText).to.be.a('function');
-        });
-      }
+  // Find all config/form.js or config/form.jsx files within src/applications
+  const configFiles = find.fileSync(
+    /config\/form\.js.?$/,
+    path.join(root, './applications'),
+  );
+
+  Object.values(configFiles).forEach(configFilePath => {
+    it(`${configFilePath.replace(root, '')}:`, () => {
+      return expect(
+        // Dynamically import the module and perform tests on its default export
+        import(configFilePath).then(({ default: formConfig }) => {
+          validFormId(formConfig);
+          validNumberProperty(formConfig, 'version');
+          validMigrations(formConfig);
+          validObjectProperty(formConfig, 'chapters');
+          validObjectProperty(formConfig, 'defaultDefinitions');
+          validFunctionProperty(formConfig, 'introduction', false);
+          validBooleanProperty(formConfig, 'prefillEnabled', false);
+          validFunctionProperty(formConfig, 'prefillTransformer', false);
+          validStringProperty(formConfig, 'trackingPrefix');
+          validTitle(formConfig);
+          validStringProperty(formConfig, 'subTitle', false);
+          validStringProperty(formConfig, 'urlPrefix', false);
+          validStringProperty(formConfig, 'submitUrl', false);
+          validFunctionProperty(formConfig, 'submit', false);
+          validObjectProperty(formConfig, 'saveFormMessages', false);
+          validFunctionProperty(formConfig, 'transformForSubmit', false);
+          validFunctionProperty(formConfig, 'confirmation');
+          validObjectProperty(formConfig, 'preSubmitInfo', false);
+          validFunctionProperty(formConfig, 'footerContent', false);
+          validFunctionProperty(formConfig, 'getHelp', false);
+          validFunctionProperty(formConfig, 'errorText', false);
+          return true;
```

Comment:
this is needed for the `).to.eventually.be.ok;` a few lines down.  If any of the expects in the above functions fail the test for the `configFilePath` fails as expected